src/highdicom/sr/templates/tid3700.py

```diff
@@ -0,0 +1,1213 @@
+                raise TypeError(
+                    'Argument "systolic_blood_pressure" must have ' +
+                    'type PressureUnit.'
+                )
+            systolic_blood_pressure.name = codes.SCT.SystolicBloodPressure
+            systolic_blood_pressure.add_items(content)
+        if diastolic_blood_pressure is not None:
+            if not isinstance(diastolic_blood_pressure, PressureUnit):
+                raise TypeError(
+                    'Argument "diastolic_blood_pressure" must have ' +
+                    'type PressureUnit.'
+                )
+            diastolic_blood_pressure.name = codes.SCT.DiastolicBloodPressure
+            diastolic_blood_pressure.add_items(content)
+        if patient_state is not None:
+            if not isinstance(patient_state, (CodedConcept, Code)):
+                raise TypeError(
+                    'Argument "patient_state" must have type ' +
+                    'Code or CodedConcept.'
+                )
+            patient_state_item = CodeContentItem(
+                name=codes.DCM.PatientState,
+                value=patient_state,
+                relationship_type=RelationshipTypeValues.CONTAINS
+            )
+            content.append(patient_state_item)
+        if pacemaker_in_situ is not None:
+            if not isinstance(pacemaker_in_situ, (CodedConcept, Code)):
+                raise TypeError(
+                    'Argument "pacemaker_in_situ" must have type ' +
+                    'Code or CodedConcept.'
+                )
+            pacemaker_in_situ_item = CodeContentItem(
+                name=CodedConcept(
+                    value='441509002',
+                    meaning='Pacemaker in situ',
+                    scheme_designator='SCT'
+                ),
+                value=pacemaker_in_situ,
+                relationship_type=RelationshipTypeValues.CONTAINS
+            )
+            content.append(pacemaker_in_situ_item)
+        if icd_in_situ is not None:
+            if not isinstance(icd_in_situ, (CodedConcept, Code)):
+                raise TypeError(
+                    'Argument "icd_in_situ" must have type ' +
+                    'Code or CodedConcept.'
+                )
+            icd_in_situ_item = CodeContentItem(
+                name=CodedConcept(
+                    value='443325000',
+                    meaning='ICD in situ',
+                    scheme_designator='SCT'
+                ),
+                value=icd_in_situ,
+                relationship_type=RelationshipTypeValues.CONTAINS
+            )
+            content.append(icd_in_situ_item)
+        if len(content) > 0:
+            item.ContentSequence = content
+        super().__init__([item])
+
+
+class PriorECGStudy(Template):
+    """:dcm:`TID 3702 <part16/sect_TID_3702.html>`
+    Prior ECG Study
+    """
+
+    def __init__(
+        self,
+        comparison_with_prior_study_done: Union[CodedConcept, Code],
```

Comment:
The allowed values for this [CID](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/sect_CID_230.html) are just "Yes", "No", and "Undetermined". Maybe to reduce boilerplate, we just allow users to pass a bool here and construct the corresponding code if they do?